pkg/api/meta/meta.go

```diff
@@ -337,6 +443,7 @@ type genericAccessor struct {
 	deletionTimestamp **unversioned.Time
 	labels            *map[string]string
 	annotations       *map[string]string
+	ownerReferences   sliceAccessor
```

Comment:
The semantics of `sliceAccessor` don't seem to be needed anymore. This can probably be replaced with `*[]metatyps.OwnerReference` and a lot of the reflection below could be avoided.